### PR TITLE
Updated hosts_vars for OFN Italy. Changed dina key.

### PR DESCRIPTION
**Pull request by kabhotz**
Updated ofn-install configuration to deploy on Ubuntu 18.04:
- changed dependencies to compile brotli
- modified compile options for brotli
- updated nginx modules path

Minor changes for OFN Italy:
- changed dina key
- removed host_vars staging.pixelwise.it
- added host_vars stg.openfoodnetwork.it

### CODE REVIEW

File: roles/brotli_nginx/defaults/main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 nginx_brotli_version: v0.1.2
-nginx_modules_path: /etc/nginx/modules
+nginx_modules_path: /usr/lib/nginx/modules
```
Comment (Matt-Yorkley) [inline roles/brotli_nginx/defaults/main.yml:4]: I don't think we want to include either of these these nginx installation changes. The original issue should be resolved by this PR: https://github.com/openfoodfoundation/ofn-install/pull/625. I manually retested it this morning on both Ubuntu 16 and 18.

Comment (sauloperez) [reply roles/brotli_nginx/defaults/main.yml:4]: ping @kabhotz

**Comment by RachL** [comment]
hello @kabhotz just making sure you saw that there are pending comments from reviewer here. Are you able to make the requested changes?

**Comment by kabhotz** [comment]
Created a new PR #692 with the confirmed modifications.

**Pull request closed without merging**